Test/Altinn.Broker.Tests/FileControllerTests.cs

```diff
@@ -335,6 +323,18 @@ public async Task UploadFile_ChecksumSetWhenInitialized_SameChecksumSetAfterUplo
         Assert.Equal(checksum, fileDetails.Checksum);
     }
 
+    [Fact]
+    public async Task SendFile_UsingUnregisteredUser_Fails()
+    {
+        var file = FileInitializeExtTestFactory.BasicFile();
```

Comment:
Consider formatting for readabiliity, splitting up in // Arrange, // Act, // Assert

Reply:
Fixed